scripts/run_server.py

```diff
@@ -21,6 +20,11 @@
 FLAGS = flags.FLAGS
 
 
+def run_one_server(cls, cfg):
```

Comment:
i would add port=None as argument, this way we can also use this function on line 37